src/api/spec/controllers/webui/attributes_spec.rb

```diff
@@ -16,4 +16,20 @@
       expect(flash[:error]).to eq("Project not found: Does:Not:Exist")
     end
   end
+
+  describe 'POST #create' do
+    let(:attribute_type) { create(:attrib_type_with_namespace, value_count: 1) }
+
+    before do
+      login user
+      post :create, params: { attrib: { project_id: user.home_project.id, attrib_type_id: attribute_type.id } }
+    end
+
+    it {
```

Comment:
Do not use `{...}` for multi-line blocks: https://github.com/bbatsov/ruby-style-guide#single-line-blocks